var/spack/repos/builtin/packages/py-cython/package.py

```diff
@@ -9,9 +9,11 @@
 class PyCython(PythonPackage):
     """The Cython compiler for writing C extensions for the Python language."""
 
+    homepage = "https://github.com/cython/cython"
```

Comment:
http://cython.org/ is listed as the homepage on PyPI

Reply:
I have a general question about that: As I see it there are good arguments for both, using the homepage listed on PyPI and using the github page of the project. From a package building view, I find it more helpful to have the github one. In PyPI this is solved by the "Sources Code" entry but in spack we don't have something like this as far as I know (that's why I set it as homepage). So should I still change it (I am asking because you already approved the PR)?

Reply:
You don't have to change it. If you want both, I would maybe add a `git` attribute for the GitHub source and use `homepage` for the actual homepage. Some packages also have documentation (i.e. Read the Docs) and we don't have a great way to include that.